packages/pwa-kit-runtime/src/ssr/server/build-remote-server.test.js

```diff
@@ -30,3 +72,16 @@ describe('remote server factory test coverage', () => {
         expect(endpoint).toBeDefined()
     })
 })
+
+describe('extensions', () => {
+    test('can register extensions properly via _setupExtensions', () => {
```

Comment:
A couple of ideas for testing:

- Test when the extension is missing.
- Test to verify that extensions are isolated from other extensions throwing an error.

Reply:
VERY GOOD CALL. added the test cases!

Reply:
it might also be a good idea to log down the behaviour for 2 extensions that have a similarly pathed handler, which one gets called? 

I'm also open to having another ticket for this as there are probably more scenarios that we want to cover with respect to the infinite list of things people will do with that express app.